core/services/keystore/keys/ocr2key/key_bundle.go

```diff
@@ -27,6 +27,7 @@ type KeyBundle interface {
 	Unmarshal(b []byte) (err error)
 	Raw() Raw
 	OnChainPublicKey() string
+	NaclBoxOpenAnonymous(ciphertext []byte) (plaintext []byte, err error)
```

Comment:
Add a comment explaining which of the many keypairs in the `KeyBundle` is used to open the ciphertext?